### DISCUSSION TITLE
Idle multiple accounts [Frequently Asked]

### DISCUSSION THREAD

**Discussion by d0biar** [Refloow/Steam-Games-Idle#4] [Q&A] [answered]
"How to idle multiple accounts"
"Add option to idle multiple accounts"
"How can i idle 100 accounts"
"How to idle more then 1 account"

**Comment by Refloow** [comment, marked as answer]
### **Support for Idling multiple accounts wont be added to the script.**
 **_Point of the script is idling one game as it would opened normally on your pc but without using any computing resources_**

You can run the script on multiple accounts by making copies of the script, but script is built to work on one and only one account, even tho you could copy and paste the code or run the script multiple times. We wont provide an integrated way to do so unless you know how to code, which we cant stop you for making any modifications, its legal to do so, but keep in mind that responsibility of using or modifying the software is yours.

Unlimited amount of accounts can create spam on the platform and potentially faking number of active players on smaller games which wont be acceptable and may applied as cheating since it gives an advantage compared to other developers/games.

**Adding modifications that make you gain any type of advantage or modifications for usage of community market, causing the spam on the platform in any way might make the platform to take actions.**

Feel free to checkout our [User Agreement](https://github.com/OSL-Works/Steam-Games-Idle/blob/master/.github/USER_AGREEMENT.md)
And project's [open source license](https://github.com/OSL-Works/Steam-Games-Idle/blob/master/LICENSE)
